ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -388,26 +375,32 @@ func canEnableContainerInstanceTagging(client ecsclient.ECSClient) (bool, error)
 	return false, nil
 }
 
-func determineArchitecture(cfnParams *cloudformation.CfnStackParams) (string, error) {
-	architecture := amimetadata.ArchitectureTypeX86
+func retrieveInstanceType(cfnParams *cloudformation.CfnStackParams) (string, error) {
+	param, err := cfnParams.GetParameter(ParameterKeyInstanceType)
 
-	// a1 instances get the Arm based ECS AMI
-	instanceTypeParam, err := cfnParams.GetParameter(ParameterKeyInstanceType)
 	if err == cloudformation.ParameterNotFoundError {
-		logrus.Infof("Defaulting instance type to t2.micro")
```

Comment:
minor: I like this info message, can you re-add it somewhere

Reply:
Done! As @allisaurus suggested, I've moved it inside `retrieveInstanceType` function